Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/LATISS-Combined-Calibrations-Procedure/latiss-combined-calibrations-procedure.rst

```diff
@@ -13,30 +13,33 @@ Latiss Combined Calibrations Generation Procedure
+The verification step is currently only implemented for biases, darks, and flats. In order to run `cp_verify`_, the verification pipetask uses as input individual images and the combined image for a particular calibration type as a reference. This reference combined image may already exist in the default calibration collection ``LATISS/calib`` (a location that can be changed via the ``input_collections_verify_bias`` and similar parameters), or, the user can indicate the script to generate a new combined calibration from the individual images taken via the ``generate_calibrations`` parameter. The former is known as ``external verification``, and the latter as ``internal verification``. The two verification modes are designed to answer slightly different questions using the same tools; for ``external verification``, the question is "are the data taken with the camera consistent with what they were in the recent past?"  This covers stability ranges on the day/week/month time scales.  ``Internal verification`` asks "are the data taken with the camera consistent with themselves?", covering stability ranges on the minute/hour level. By default, the script will run in ``external verification`` mode.
+
+For at least one type of test (as defined in `DMTN-101`_), if the majority of tests fail verification in the majority of detectors and the majority of exposures, then the script will terminate after calculating these verification statistics. If ``generate_calibrations`` is ``True``, the calibration will not be certified into the ``calibration_collection`` collection. The configuration parameters ``number_verification_tests_threshold_bias``, ``number_verification_tests_threshold_dark``, and ``number_verification_tests_threshold_flat`` will be used to define thresholds to decide whether the calibration will pass verification and should be certified or not. Currently, verification is only implemented for ``BIAS``, ``DARK``, and ``FLAT`` calibration types. If the configuration parameters ``do_defects`` and ``do_ptc`` are set to ``True``, verification will be skipped for the ``DEFECTS`` and ``PTC`` calibrations and they will be automatically certified.
 
 The script currently has the option (via the ``script_mode`` parameter in the configuration options) to:
 
 - take only biases, 
```

Comment:
so when you say "take only biases" -- it sounds like you mean just taking the data, but I'm assuming that's not what you mean since you are talking about the construction, verification and certification below. 

I think you want to say something like, "- take a series of bias exposures and ultimately produce a certified bias" ?